packages/commons/src/config/db.config.ts

```diff
@@ -19,6 +19,7 @@ export const SignalHubStoreConfig = z
     SH_DB_USE_SSL: z
       .enum(["true", "false"])
       .transform((value) => value === "true"),
+    SH_MAX_CONNECTION_POOL: z.coerce.number().default(30),
```

Comment:
Does it need to add optional()? What happens if the variable is not set?

Reply:
If it's not set, applications doesn't run due to this var is on db.config.ts (so it's checking if some var missing on environment configuration). We could decide to pass default = 30 on createDbInstance signature and put .optional after .number()